scripts/jenkins/ardana/ardana-init.bash

```diff
@@ -27,9 +27,7 @@ SCRIPT_DIR="$(readlink -e $(dirname "${BASH_SOURCE}"))"
 owner="$(stat -c '%U' "$SCRIPT_DIR")"
 group="$(stat -c '%G' "$SCRIPT_DIR")"
 
-deployer_version="ardana-0.9.0"
-ansible_package_dir="/opt/stack/venv/ansible-1.9.3"
-ansible_service_dir="/opt/stack/service/ansible-1.9.3"
+deployer_version="ardana-8"
```

Comment:
I'm fine with this change - but does it work? or do we need also ansible changes to use the new path?